src/components/App.js

```diff
@@ -23,11 +24,44 @@ class App extends React.Component {
     return (
       <Provider store={store}>
         <I18nextProvider i18n={i18n}>
-          <Router
-            history={browserHistory}
-            children={routes}
-            render={applyRouterMiddleware(useScroll())}
-          />
+          <div>
+            <Helmet defaultTitle='inspire.data.gouv.fr'>
```

Comment:
you can add a `titleTemplate` here like `'%s | inspire.data.gouv.fr'`, so when in sub components you set `title='Foo'`, the title will be `<title>Foo | inspire.data.gouv.fr</title>`